content/millennium/dstu2/care-provision/care-plan.md

```diff
@@ -32,6 +34,9 @@ _Implementation Notes_
  `patient`    | This, or one of `_id` or `subject`     | [`reference`] | Who care plan is for. Example: `patient=1316024`
  `subject`    | This, or one of `_id` or `patient`     | [`reference`] | Who care plan is for. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/1316024` or `subject:Patient=1316024`
  [`_count`]   | N                                      | [`number`]    | Number of results per page. Defaults to `10`. Capped at `10`.
+ `category`   | N                                      | [`token`]     | The scope of care plan being searched for. Examples: `category=assess-plan`, `category=http://argonaut.hl7.org|careteam`, `category=http://hl7.org/fhir/care-team-category|longitudinal`, or `category=careteam`
+ `context`    | N                                      | [`reference`] | The encounter id of the care plan. Example: `context=1138`
+
 
 Notes:
 
```

Comment:
"When providing a category, `_id` will only allow a code of `assess-plan`."  doesn't make sense to me.  I propose:
```
- The `_id` parameter may not be provided at the same time as the `date`, `patient`, or `subject` parameters.
  - When `_id` is provided, `_count` is ignored.
  - When `_id` is provided, `category`, if set, must be the code `assess-plan`.
```